packages/console/src/pages/Connectors/components/ConnectorForm/index.tsx

```diff
@@ -102,16 +103,18 @@ const ConnectorForm = ({ connector, isAllowEditTarget }: Props) => {
           )}
         />
       </FormField>
-      <FormField isRequired title="connectors.guide.sync_profile">
-        <Controller
-          name="syncProfile"
-          control={control}
-          rules={{ required: true }}
-          render={({ field: { onChange, value } }) => (
-            <Select options={syncProfileOptions} value={value} onChange={onChange} />
-          )}
-        />
-      </FormField>
+      {connector.type === ConnectorType.Social && (
+        <FormField isRequired title="connectors.guide.sync_profile">
```

Comment:
this `isRequired` is redundant in `<Select />` component.

Reply:
rebased